src/components/UI/Overlay/OverlayContent/DocumentTransferMessage.tsx

```diff
@@ -41,17 +42,48 @@ const ButtonMetamaskInstall = () => {
   );
 };
 
+const ButtonConfirmAction = (handleOnClick: () => void) => {
+  const { setOverlayVisible } = useContext(OverlayContext);
+  const onClick = () => {
+    handleOnClick();
+    setOverlayVisible(false);
+  };
+  return (
+    <ButtonSolidOrangeWhite onClick={onClick} data-testid={"confirmActionBtn"}>
+      Confirm
+    </ButtonSolidOrangeWhite>
+  );
+};
+
 interface DocumentTransferMessageProps extends OverlayContentProps {
   children: React.ReactNode;
   isButtonMetamaskInstall?: boolean;
+  isConfirmationMessage?: boolean;
+  onConfirmaionAction?: () => void;
 }
 
 export const DocumentTransferMessage = styled(
-  ({ isButtonMetamaskInstall, children, ...props }: DocumentTransferMessageProps) => {
+  ({
+    isButtonMetamaskInstall,
+    isConfirmationMessage,
+    onConfirmaionAction,
```

Comment:
```suggestion
    onConfirmationAction,
```

Reply:
The rest all typo also

Reply:
![image](https://user-images.githubusercontent.com/7406870/95428314-7300ed00-097b-11eb-89d6-1785759c9508.png)

Suggest a spell checker extension. Laurent also has one running in his IntelliJ